js/components/graph/Graph.js

```diff
@@ -160,7 +160,9 @@ export class Graph extends React.Component {
           } else if (entry.type_ === "Hybrid") {
             hybridsList.push(entry)
           } else if (entry.type_ === "BoolNode") {
-            boolsStatusObj[entry.id_] = "inactive"
+            boolsStatusObj[entry.id_] = localStorage.getItem(entry.id_)
```

Comment:
It's a common pattern to use `x || y` instead of `x ? x : y`, taking advantage of the `||` short-circuiting behaviour (and the fact that it actually returns one of `x`/`y`, not a boolean).